livingatlas/scripts/la-pipelines

```diff
@@ -313,7 +320,7 @@ function interpret () {
             $(toArg $PRE driver-memory) \
             --class $CLASS \
             --master $SPARK_MASTER \
-            --driver-java-options "-Dlog4j.configuration=file:/efs-mount-point/log4j.properties" \
+            --driver-java-options "-Dlog4j.configuration=file:$CONFIG_DIR/log4j.properties" \
```

Comment:
I think we should use `$logConfig` better, but, now I remember that you commented me that you have issues with colorized logs running in Spark caused by a Class not found exception.

Some related discussion:
https://stackoverflow.com/a/52740534
https://stackoverflow.com/a/47734870

So me merge this like this, and also we can open an additional issue and try to fix it using `$logConfig`.

Reply:
Ive switched to using $logConfig and this is working fine.